api/java/aggregation/ungroup.md

```diff
@@ -116,9 +109,21 @@ Result: (Note this is a JSON representation of a `List<GroupedResult>`; see the
             {"id": 11, "player": "Bob", "points": 10, "type": "free"}
         ]
     }
-}
+[
 ```
 
+__Example:__ Finding the statistical mode of an array of values:
+```java
+r.expr(r.array([1,2,2,2,3,3])).group(
+  row -> row
+).ungroup().orderBy("reduction").nth(-1).bracket("group")
```

Comment:
Two more things about this example:
1. It's missing the `count()`
2. Instead of `bracket` we should probably use `getField` for better readability.